query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -2,20 +2,21 @@ package influxdb
 
 import (
 	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
 	"github.com/influxdata/flux/execute"
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/semantic"
 	"github.com/influxdata/flux/stdlib/universe"
 )
 
-// func init() {
-// 	plan.RegisterPhysicalRules(
-// 		PushDownRangeRule{},
-// 		PushDownFilterRule{},
-// 		PushDownReadTagKeysRule{},
-// 		PushDownReadTagValuesRule{},
-// 	)
-// }
+func init() {
+	plan.RegisterPhysicalRules(
+		PushDownRangeRule{},
+		PushDownFilterRule{},
+		//PushDownReadTagKeysRule{},
```

Comment:
I thought we can un-comment this one too?

Reply:
Its merged to master but not deployed yet. So I wanted to hold off on that one so we can deploy this change without blocking on a storage deploy.